fakebook/src/app/service/notifications.service.ts

```diff
@@ -65,4 +69,26 @@ export class NotificationsService {
   setRead(ids: string[]): void {
     this.hubConnection.invoke('MarkNotificationAsReadAsync', ids);
   }
+
+  // invokes the Create comment function on the signalR backend
+  createCommentNotification(commenterEmail: string, postId: number): void {
+
+    let posterEmail: string;
+
+    // gets email of the post creator and sends a new notification to the backend
+    this.postService.getById(postId).subscribe(post => {
+
+      posterEmail = post.userEmail;
+
+      const notif: ApiNotification = {
+        id: '',
+        date: new Date(Date.now()),
+        triggerUserId: commenterEmail,
+        loggedInUserId: posterEmail,
+        type: {key: 'comment', value: 0},
+        hasBennread: false
```

Comment:
there's still a typo in this model. it's probably working fine because the bool in .net will default to false anyway.